cli/cmd/lql.go

```diff
@@ -454,7 +459,13 @@ func runAdhocQuery(cmd *cobra.Command, args []api.ExecuteQueryArgument) (
 		return
 	}
 
-	cli.StartProgress(" Executing query...")
+	msg := "Executing query"
+	startTime, startErr := time.Parse(time.RFC3339, args[0].Value)
+	endTime, endErr := time.Parse(time.RFC3339, args[1].Value)
+	if startErr == nil && endErr == nil {
+		msg = fmt.Sprintf("%s in the time range %s - %s", msg, startTime.Format("2006-Jan-2 15:04:05 MST"), endTime.Format("2006-Jan-2 15:04:05 MST"))
```

Comment:
nit
```suggestion
		msg = fmt.Sprintf("%s within time range %s - %s", msg, startTime.Format("2006-Jan-2 15:04:05 MST"), endTime.Format("2006-Jan-2 15:04:05 MST"))
```

Reply:
I think either wording is fine, will leave as is.